resources/js/views/Statistics.vue

```diff
@@ -55,6 +57,8 @@ const load = ref(false);
 
 const { is_se_preview_enabled, are_nsfw_visible } = storeToRefs(lycheeStore);
 
+const showTotal = computed(() => total.value !== undefined && (total.value.num_albums > 0 || total.value?.num_photos > 0 || total.value?.size > 0));
```

Comment:
Are the `?`s required here?

Reply:
Fixed [2ccf57e](https://github.com/LycheeOrg/Lychee/pull/3059/commits/2ccf57eb1b1b62aa95670e8d08d851bedeffce75)